shared/src/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -371,6 +433,44 @@ public void removeChangeListener(ListenerToken token) {
         }
     }
 
+    /**
+     * Set the given DocumentReplicatedListener to the this replicator.
+     *
+     * @param listener
+     * @return ListenerToken A token to remove the handler later
+     */
+    public ListenerToken addReplicationListener(DocumentReplicatedListener listener) {
+        return addReplicationListener(null, listener);
+    }
+
+    /**
+     * Set the given DocumentReplicatedListener to the this replicator.
+     *
+     * @param listener
+     */
+    public ListenerToken addReplicationListener(Executor executor, DocumentReplicatedListener listener) {
```

Comment:
The correct signature is `addDocumentReplicationListener(Executor executor, DocumentReplicationListener listener)`.